### ISSUE TITLE
None but ObjectFactory class created

### ISSUE THREAD

**Issue by soberich** [SixRQ/KAXB#2]
Hi,
I understand the project is abandoned but here is the output generated with whatever I tried.
(P.S. XSDs are 100% correct since JAXB Ajc and other tools produces proper outputs)
```kotlin
package com.example

import javax.xml.bind.JAXBElement
import javax.xml.bind.annotation.XmlElementDecl
import javax.xml.bind.annotation.XmlRegistry
import javax.xml.namespace.QName

/**
 * This object contains factory methods for each
 * Kotlin content interface and Kotlin element interface
 * generated in the com.example package.
 * <p>An ObjectFactory allows you to programatically
 * construct new instances of the Kotlin representation
 * for XML content. The Kotlin representation of XML
 * content can consist of schema derived interfaces
 * and classes representing the binding of schema
 * type definitions, element declarations and model
 * groups.  Factory methods for each of these are
 * provided in this class.
 *
 */

@XmlRegistry
class ObjectFactory {



}
```
Nothing else produced

**Comment by legzo** [comment]
Yeah, same here :/

**Comment by kenteg** [comment]
same problem

**Comment by asm0dey** [comment]
same :)

**Comment by mrlem** [comment]
Ok, I had the same behaviour, so I investigated a bit: the parser is hardcoded for xsd namespace to be "xsd" and nothing else (hence the "empty ObjectFactory" issue).

Beyond that:

- develop branch is hopelessly broken: it's actually a work in progress that generates part kotlin / part java classes (which obviously doesn't work)
- I couldn't get anything to run on master: the app hangs on unit tests

I don't have more time to waste on this, so I'll give up and look somewhere else :roll_eyes:

**Comment by asm0dey** [comment]
I've built my own extensible and fast xml parser: https://github.com/asm0dey/staks/

You're welcome to provide me with feedback or contributions :)

**Comment by mrlem** [comment]
At first glance, it wasn't my exact need (I meant to generate element classes based on an XSD or a RNC file), but I'll look into it, thanks :smiley:

**Comment by asm0dey** [comment]
Yeah, it's not about XSD; it's for simple generation of data from XML, even not well-formed.
